alloy/src/main/java/com/liferay/faces/alloy/render/internal/AlloyRendererBase.java

```diff
@@ -44,19 +46,19 @@ public void decodeClientBehaviors(FacesContext facesContext, UIComponent uiCompo
 	@Override
 	public void encodeBoolean(ResponseWriter responseWriter, String attributeName, Boolean attributeValue,
```

Comment:
@ngriffin7a, I prefer the name `encodeBooleanProperty` (I think it's clearer), which is why I changed it in Util. But this is not public API and it requires changes to the generator, so I left it alone.